### PR TITLE
update mta

### LINKED ISSUE #735: MTA:SA(GTA) fails to reinstall
Panel Version: 0.7
Daemon Version: 0.6
Service: Multi Theft Auto
Modified: No

Expected Behavior: Ability to reinstall the server with no errors

Actual Behavior: Stalls when reinstalling, at mta-resources.zip as files already exist so i think it waits for prompt to overwrite

Steps to Reproduce: Create MTA server, after installation, trigger server reinstall

### LINKED ISSUE #739: Add libmysqlclient.so.16 to ubuntu_source for multi theft auto egg
Panel Version: 0.7
Daemon Version: 0.6
Service: Multi Theft Auto
Modified: No

Expected Behavior: libmysqlclient.so.16 should be available in /usr/lib so that a popular multi theft auto gamemode(MTA-Paradise) that uses an old mysql library to load properly or maybe add flag in installation. More info here: https://forum.mtasa.com/topic/66856-libmysqlclientso16-cannot-open-shared-object-file/

Actual Behavior: Library not available so gamemode fails to start

Steps to Reproduce: Install fresh mta server, load configs for mta-paradise and mysql module, start the server and you will get an error like this: 
`MODULE: Unable to load x64/modules/mta_mysql.so (libmysqlclient.so.16: cannot open shared object file: No such file or directory)`

### PR DESCRIPTION
**Pull request by parkervcp**
resolves #735 

### All Submissions:
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:
1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [x] Have you tested your Egg changes?

This needs further testing.

### CODE REVIEW

**Comment by parkervcp** [comment]
@ciber96 can you look at this?

File: gta/mtasa/egg-multi-theft-auto.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
...
-            "container": "ubuntu:16.04",
+            "script": "#!\/bin\/bash\r\napt -y update\r\napt -y --no-install-recommends install curl unzip lib32gcc1 ca-certificates\r\n\r\ncd \/tmp\r\ncurl -L -o multitheftauto_linux_x64.tar.gz https:\/\/linux.mtasa.com\/dl\/multitheftauto_linux_x64.tar.gz\r\ncurl -L -o mta-baseconfig.tar.gz https:\/\/linux.mtasa.com\/dl\/baseconfig.tar.gz\r\ncurl -L -o mtasa-resources-latest.zip http:\/\/mirror.mtasa.com\/mtasa\/resources\/mtasa-resources-latest.zip\r\n\r\nmkdir -p \/mnt\/server\r\ntar -xzvf multitheftauto_linux_x64.tar.gz\r\ncp -rf multitheftauto_linux_x64\/* \/mnt\/server\r\n\r\nif [ ! -f \/mnt\/server\/x64\/libmysqlclient.so.16 ]; then\r\n    curl -L http:\/\/nightly.mtasa.com\/files\/libmysqlclient.so.16 -o \/mnt\/server\/x64\/libmysqlclient.so.16\r\nfi\r\n\r\nmkdir -p \/mnt\/server\/mods\/deathmatch\/resources\r\nunzip -o -d \/mnt\/server\/mods\/deathmatch\/resources mtasa-resources-latest.zip\r\n\r\nmkdir -p \/mnt\/server-conf\r\ntar -xzvf mta-baseconfig.tar.gz\r\ncp -rf baseconfig\/* \/mnt\/server\/mods\/deathmatch\r\n\r\nchown -R root:root \/mnt\r\n\r\nexport HOME=\/mnt\/server",
```
Comment (ciber96) [inline gta/mtasa/egg-multi-theft-auto.json:20]: yeah so i the server looks for that lib in /usr/lib not in `server/x64/libmysqlclient.so.16` @parkervcp

**Comment by ciber96** [comment]
Tested out reinstall and is working fine now. Thanks

**Comment by parkervcp** [comment]
@ciber96 does this also fix the mysql lib issue?

**Comment by parkervcp** [comment]
@ciber96 can you try out the egg but use `quay.io/parkervcp/pterodactyl-images:game_mtasa` as the image

**Comment by parkervcp** [comment]
@ciber96 can you validate the mysql lib thing please

**Comment by ciber96** [comment]
No, i doesnt work. Because the correct location for the lib is in: /usr/lib @parkervcp

**Comment by parkervcp** [comment]
I ask because string an let lib location should make it load libraries.

I don't know how to set to the things yours using so I need your help testing. Please understand that I have over 100 eggs and do not know how to configure and use them all @ciber96

**Comment by parkervcp** [comment]
Going to merge this at least for the reinstall issue.

**Pull request merged**